plugins/adapters/modbus/srcCxx/LibModbusClient.cxx

```diff
@@ -68,6 +68,30 @@ void LibModbusClient::disconnect()
     }
```

Comment:
Run clang-format on this file....

Opening braces on methods should be in the next line